### ISSUE TITLE
Vote-Rigging is Broken

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#35578]
At least for maps that don't meet vote requirements. If you add an extra map like horizon, it wins the vote but you go to box.

**Comment by SacredAtom** [comment]
Yeah unless you hit (all compiled) in the vote menu before map vote comes up, I found rigging the vote doesn't function usually.
